_includes/global-header.html

```diff
@@ -10,24 +10,18 @@
                     <div class="navbar-collapse collapse">
                         <ul class="primary nav navbar-nav">
                             <li><a href="https://docker.com/why-docker">Why Docker?</a></li>
-                            <li><a href="https://docker.com/get-started">Product</a></li>
+                            <li><a href="https://www.docker.com/products">Products</a></li>
                             <li class="dropdown">
                                 <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Get Docker <span class="caret"></span></a>
                                 <ul class="dropdown-menu nav-main">
                                     <h6 class="dropdown-header">For Desktop</h6>
-                                    <li><a href="https://docker.com/products/docker-desktop">Mac & Windows</a></li>
-                                    <h6 class="dropdown-header">For Cloud Providers</h6>
-                                    <li><a href="https://docker.com/partners/aws">AWS</a></li>
-                                    <li><a href="https://docker.com/partners/microsoft">Azure</a></li>
+                                    <li><a href="https://hub.docker.com/editions/community/docker-ce-desktop-mac">Mac</a></li>
+									<li><a href="https://hub.docker.com/editions/community/docker-ce-desktop-windows">Windows</a></li>
                                     <h6 class="dropdown-header">For Servers</h6>
-                                    <li><a href="https://hub.docker.com/editions/enterprise/docker-ee-server-windows">Windows Server</a></li>
-                                    <li><a href="https://hub.docker.com/editions/enterprise/docker-ee-server-centos">CentOS</a></li>
+                                    <li><a href="https://hub.docker.com/editions/community/docker-ce-server-centos">CentOS</a></li>
```

Comment:
I'm wondering what the goal is of these links. Looking at this one (for example) will link to Docker Hub, which only has a description (and the description is actually outdated), and to _install_, it gives a link back to the documentation (https://docs.docker.com/engine/installation/linux/centos/)